src/App.js

```diff
@@ -4,7 +4,7 @@ import {Home} from "./pages/homePage/Home"
 import { Routes, Route } from "react-router-dom";
 import {ProductListing} from "./pages/productListing/ProductListing";
```

Comment:
It's better to create a index.js file in your pages folder where you import all individual jsx pages and export, it'll shorten your path here as you can just write import...from ".....pages"